src/keyboard-layout-manager-windows.cc

```diff
@@ -151,7 +151,7 @@ Local<Value> CharacterForNativeCode(HKL keyboardLayout, UINT keyCode, UINT scanC
   wchar_t characters[5];
   int count = ToUnicodeEx(keyCode, scanCode, keyboardState, characters, 5, 0, keyboardLayout);
 
-  if (count > 0 && !std::iscntrl(characters[0])) {
+  if (count > 0 && !std::iswcntrl(characters[0])) {
     return Nan::New<String>(reinterpret_cast<const uint16_t *>(characters), count).ToLocalChecked();
```

Comment:
Is this cast still safe? `wchar_t` is either 32-bit or 16-bit depending on the system (I believe it's always 16-bit on Windows).